### PR TITLE
docs: Updated RedwoodJS Quick Start with Prisma 

### PR DESCRIPTION
**Pull request by dthyresson**
## What kind of change does this PR introduce?

This PR follows the basic outline of the other framework QuickStarts to create and populate and list a handful of Countries:

* Adds a RedwoodJS Framework Quick Start that showcases how to use RedwoodJS with Prisma and Supabase as its Postgres database. It describes how to create a project, configure Redwood and Prisma, use Prisma migrations and seeds to creamer and populate the schema and then build a CRUD UI scaffold to manage Countries.

* Syncs up the menu of QuickStarts with the content cards. Some items were missing.

## What is the current behavior?

Currently, there is no RedwoodJS framework quickstart, only the previous Guide aka Web Tutorial (which will also be updated).

### All Quickstarts listed

Screenshot 2023-04-05 at 7 39 17 AM

## RedwoodJS Quickstart

Screenshot 2023-04-05 at 7 37 13 AM
Screenshot 2023-04-05 at 7 37 21 AM
Screenshot 2023-04-05 at 7 37 29 AM
Screenshot 2023-04-05 at 7 37 36 AM
Screenshot 2023-04-05 at 7 42 56 AM

## Additional context

This PR is for Supabase Launch Week 7.

Note: the Web Tutorial is next to be updated and will have more advanced setup, such as how to use RLS with Prisma.

### CODE REVIEW

**Comment by dthyresson** [comment]
I just noticed "Start the App" was duplicated in the heading. Fixing.

File: apps/docs/pages/guides/getting-started/quickstarts/redwoodjs.mdx [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+      DIRECT_URL="postgres://postgres:[YOUR-PASSWORD]@db.[YOUR-PROJECT-REF].supabase.co:5432/postgres"
+      # PostgreSQL connection string with pgBouncer config — used by Prisma Client
+      DATABASE_URL="postgres://postgres:[YOUR-PASSWORD]@db.[YOUR-PROJECT-REF].supabase.co:6543/postgres?pgbouncer=true"
+
+      ```
+    </StepHikeCompact.Code>
+
+  </StepHikeCompact.Step>
+
+  <StepHikeCompact.Step step={5}>
```
Comment (gregnr) [inline apps/docs/pages/guides/getting-started/quickstarts/redwoodjs.mdx:120]: Looks like we got two step 5's here 👀

**Review by gregnr — DISMISSED** [review]
Great, thanks for putting this together @dthyresson! 😃 Just a minor fix and we can merge this.

**Comment by dthyresson** [comment]
> Great, thanks for putting this together @dthyresson! 😃 Just a minor fix and we can merge this.

Thanks! I renumbered the steps and also cleaned up the whitespace and formatting (am used to having prettier setup and only now realized my indentation was off everywhere).

I still would like the redwoodJS team to review, but I think it's good to merge and we can always update and improve.

**Review by kiwicopple — APPROVED** [review]

**Pull request merged**